babel/test/evaluate.js

```diff
@@ -0,0 +1,73 @@
+import test from 'ava';
+import * as _ from 'lodash';
+import {names} from './data/names';
+import {orbs} from './data/orbs';
+import {testMatchData} from './data/evaluate';
+import {nonMatchDropped} from './data/evaluate';
+import {droppedRandoms} from './data/evaluate';
+import {unaffectedOrbs} from './data/evaluate';
+import {evaluate} from '../src/board';
+import {findMatches} from '../src/board';
+import {combineMatches} from '../src/board';
+import {Board} from '../src/board';
+let board;
+let evaluatedOrbs;
+let matchDatas;
+let name;
+
+test.before(() => {
+    board = new Board(5, 5);
+    evaluatedOrbs = [];
+    matchDatas = [];
+
+    _.each(_.range(orbs.length), i => {
+        board.orbs = orbs[i];
```

Comment:
It also made me wonder if this spot is affect by this concept. I think it is affected, but it doesn't matter since I'm not referencing  `orbs[i]` again. It still feels like I should change this to `board.orbs = _.cloneDeep(orbs[i]);` since that is what I'm doing.